packages/frames.js/src/getFrame.ts

```diff
@@ -24,6 +24,12 @@ type GetFrameOptions = {
    * @defaultValue 'farcaster'
    */
   specification?: SupportedParsingSpecification;
+  /**
+   * If true, a warning will be reported if the title is missing.
+   *
+   * @defaultValue false
+   */
+  warnOnMissingTitle?: boolean;
```

Comment:
is this option necessary?

Reply:
It is unfortunately because otherwise you would get warnings of missing title on any frame, for example for POST requests.

Reply:
Also it is for backwards compatibility so it doesn't start to trigger warnings out of nowhere. But I'm not sure if that should actually start reporting by default or not, so I'm open to change it if that makes sense to you, otherwise I would keep it as is.

Reply:
perhaps it makes more sense to add a request method param and warn based on that?

Reply:
same thing applies to og:image technically, we just always warn about that rn iirc

Reply:
I think it is weird to define anything about request since `getFrame()` doesn't know where the html is coming from. But it also makes sense to rename it so it also covers `og:image` reporting.

Reply:
maybe just say whether the frame is initial or a response to interaction? Something like `intent: initial | response` ?

Reply:
Ok I think actually `fromRequestMethod` makes sense.

Reply:
@stephancill Ok I used `fromRequestMethod` so it better communicates the intent of the option, `requestMethod` would imply that the `getFrame()` actually does a request itself.

Reply:
it also removes `og:image` warning for POST requests